taskcluster/ffios_taskgraph/transforms/secrets.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file is a copy of https://github.com/mozilla-mobile/fenix/blob/24596fd3cd493f38e1212e6ab2844e723565af9b/taskcluster/fenix_taskgraph/transforms/secrets.py. No change.